modules/social_features/social_content_block/social_content_block.install

```diff
@@ -426,3 +426,17 @@ function social_content_block_update_8801(&$sandbox) {
   $field_purchasers->setSetting('allowed_values', $allowed_values);
   $field_purchasers->save();
 }
+
+/**
+ * Add translations support for "Custom content list" custom block fields.
+ */
+function social_content_block_update_10200() {
```

Comment:
Since we add this to 10.3 milestone, need to rename the number to start with 103

Reply:
fixed